operator-common/src/main/java/io/strimzi/operator/common/AbstractOperator.java

```diff
@@ -339,4 +341,16 @@ public Counter getPeriodicReconciliationsCounter() {
     public AtomicInteger getResourceCounter() {
         return resourceCounter;
     }
+
+    public String getLoggingAppenders(String loggingConfiguration) {
```

Comment:
is loggingAppenders intended to be a common interface across all operators?

What about applications that don't use log4j?

Reply:
It is just common method to get appenders from the log4j configuration.

Reply:
Does it really need to be in `AbstractOperator` though? It seems like a utility method to me.